package.json

```diff
@@ -3,7 +3,7 @@
     "version": "0.10.12",
     "private": true,
     "scripts": {
-        "start": "webpack-dev-server --port=4200 --host 0.0.0.0 --open",
```

Comment:
But I thought the host flag was needed to access the demo app on VMs. Why have we removed that? Also what does the `-d` flag do?